### ISSUE TITLE
CSV not adding Newline

### ISSUE THREAD

**Issue by hutiwephy** [jjuback/gc-excelviewer#180]
when adding a new row the editor doesn't add a newline on the source code resulting in new columns instead of a new row

**Comment by hutiwephy** [comment]
OS: windows 10 (x86_64)
VSCode Version: Latest

Base CSV Source Code:
```
     Emission Date, Face Value, Country, "Description"
     2008-11-05, 1.65, QWERTY, "Very ugly"
```
Result:
```
     Emission Date, Face Value, Country, "Description"
     0000-00-00,0,Place2,"Unknown"2008-11-05, 1.65, QWERTY, "Very ugly"
```
Intended Result:
```
     Emission Date, Face Value, Country, "Description"
     0000-00-00, 0, Place2, "Unknown"
     2008-11-05, 1.65, QWERTY, "Very ugly"
```

**Comment by jjuback** [comment]
Fixed in pre-release version 4.1.52.
